build.assets/version-check/main.go

```diff
@@ -0,0 +1,151 @@
+/*
```

Comment:
is this meant to be in this PR? Could it be pulled out into its own?

Reply:
I was about to ask about the same.